web/themes/custom/pineapple/components/_heading.twig

```diff
@@ -0,0 +1,10 @@
+{% set heading_attributes = heading_attributes|default(create_attribute()) %}
+{% set heading_level = heading_level|default('3') %}
+
+{% if heading_level == '1' %}
+  <h1 {{ heading_attributes }}>{{ heading }}</h1>
+{% elseif heading_level == '2' %}
+  <h2 {{ heading_attributes }}>{{ heading }}</h2>
+{% else %}
+  <h3 {{ heading_attributes }}>{{ heading }}</h3>
+{% endif %}
```

Comment:
Instead of writing loads of if statements you can do `<h{{ heading_level }}>` then the same on the closing tag so that it can work for any size heading

Reply:
I tried that, it doesn't work. I tried it every other way I could thing of and it did not work or broke the code. This is the best I can do.

Reply:
I've tried that before I committed this atrocious code and it didn't work as desired. The issue was using quotes around integers, which I didn't realise was not necessary. Rectifying this issue allowed me to write this code as you suggested.